USAGE.md

```diff
@@ -19,76 +19,125 @@ Now `make install` only supports Linux Ubuntu, CentOS and Mac OS X. If you use o
 you should follow [INSTALL.md](https://github.com/facebook/rocksdb/blob/master/INSTALL.md) to
 install RocksDB manually first. 
 
-### Running in the standalone mode
+### Running in the local standalone mode
 
-1. Start etcd on the default listening port 2379.
 
-    ```sh
-    etcd 
-    ```
-
-2. Start PD on listening port 1234 and cluster ID is 1.
+1. Start PD.
 
     ```sh
-    pd-server -addr 127.0.0.1:1234 --etcd 127.0.0.1:2379 --cluster-id 1 --root /pd
+    pd-server --cluster-id=1 \
+              --addr=127.0.0.1:1234 \
+              --advertise-addr="127.0.0.1:1234" \
+              --http-addr="127.0.0.1:9090" \
+              --etcd-name="default" \
```

Comment:
no, you can see the etcd doc https://github.com/coreos/etcd/blob/master/Documentation/op-guide/clustering.md , it also use so many arguments. 

And the arguments should be listed here because we need to let user know how to change it.